services/ImageService_test.go

```diff
@@ -713,9 +717,6 @@ var _ = ginkgo.Describe("Image Service", func() {
 			ginkgo.It("should return that image has no ID", func() {
 				gomega.Expect(err).Should(gomega.BeNil()) // no error during BeforeEach
 
-				permRequest := requests.CreatePermissionRequestBuilder().Allow(requests.User,
```

Comment:
You can't just delete the assignment and rely on the previous test (`It`) setting it. In case the test are not run in the same order, or test is run separately, it will fail. The rule of thumb here is to use a common `Before` block and set the variable to the most common value. For test that need a different value you have no other choice then set them for that test separately.